ui/app/src/components/DependencySelection.tsx

```diff
@@ -324,17 +324,8 @@ export function DependencySelectionDelete(props: DependencySelectionProps) {
 }
 
 function SelectionList(props: SelectionListProps) {
-  const {
-    title,
-    subtitle,
-    items,
-    uris,
-    onItemClicked,
-    onSelectAllClicked,
-    displayItemTitle,
-    checked,
-    setChecked
-  } = props;
+  const { title, subtitle, items, uris, onItemClicked, onSelectAllClicked, displayItemTitle, checked, setChecked } =
+    props;
```

Comment:
Formatting doesn't look right.

Reply:
I ran prettier in the file and it says that is properly formatted